x/gov/abci.go

```diff
@@ -58,8 +58,9 @@ func EndBlocker(ctx sdk.Context, keeper *keeper.Keeper) {
 
 		if passes {
 			var (
-				idx int
-				msg sdk.Msg
+				idx    int
+				events sdk.Events
+				msg    sdk.Msg
```

Comment:
Why do we need to declare `idx` and `msg`?

Reply:
Looks like all they're used for is building a log msg below in the case of failure:

`logMsg = fmt.Sprintf("passed, but msg %d (%s) failed on execution: %s", idx, sdk.MsgTypeURL(msg), err)`

If they weren't declared they would only be available inside the scope of the `for`.